_test/test_data_loaders/test_asciipar_loader.m

```diff
@@ -77,7 +77,7 @@ function test_load_ASCII_par_binary(obj)
             if ispc
                 assertEqual([fname,fext],obj.test_par_file);
             else
-                assertEqual([fname,fext],obj.test_par_file);
+                assertEqual([fname,fext],'demo_par.PAR');
```

Comment:
is the test file `demo_par.PAR` or `demo_par.par`? You should be able to use the correctly-cased name on Linux and Windows.

Reply:
The code makes files on linux behave like the files on Unix. This was the user problem for the users who were taking random case files from Windows and complaining that they do not work on Linux. So, it is the test for picking mixed case files on any operating system

Reply:
what is the purpose of this test:
i) that a Windows machine can read 'demo.par' and 'demo.PAR' 
ii) that a Linux machine can read files named 'demo.par' and 'demo.PAR' 

If that is the case we should have two tests: one reading `demo.par`, one reading `demo.PAR` executed on both OSs